### ISSUE TITLE
Decreasing Learning Rate Improves Convergence

### ISSUE THREAD

**Issue by NickShahML** [igul222/improved_wgan_training#25]
Hey @igul222, thanks again for releasing this code. Has really helped me experiment with WGANs. I noticed in your paper protocol that you do NOT decrease the learning rate over 200k iterations.

I have found however, that after these first 200k iterations, if you decrease the learning rate of the discriminator it can find lower minima that defines a more accurate wasserstein distance. Was there a reason why you guys did not do this in your paper? Does this break any theoretical assumptions?

**Comment by igul222** [comment]
We didn't use learning rate decay for sake of simplicity in the paper, but
I think it's probably a good idea in practice! We have a few experiments
we're working on where we use it and it does seem to help help.

**Comment by igul222** [comment]
(You probably also want to decrease the learning rate of the generator at
the same time, to avoid it getting ahead too much)

**Comment by NickShahML** [comment]
Thanks for the tip about decreasing the learning rate of the generator. I'll experiment more with those methods. I've also experimented with training for more than 200k iterations and have found that given enough critic iterations (like 50), you can also achieve a better convergence.
